tss-esapi/src/interface_types/algorithm.rs

```diff
@@ -237,6 +237,7 @@ impl TryFrom<TPMI_ALG_SYM> for SymmetricAlgorithm {
 pub enum SymmetricMode {
     Ctr,
     Ofb,
+    Cbc,
```

Comment:
This is definitely a breaking change - if it's in a different commit then it's alright. I'll try and think of a way to check whether the changes in a PR relate to breaking additions.